fabric-api-dev/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,28 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-api-dev",
+  "name": "Fabric API Dev Utilities",
```

Comment:
Im not happy with this name, it doesnt explain what it does. I dont have a good suggestion otherwise though.